src/components/UnstyledButton/UnstyledButton.tsx

```diff
@@ -58,6 +61,9 @@ export function UnstyledButton({
   };
   const interactiveProps = {
     ...commonProps,
+    role: loading
+      ? [...new Set([ARIA_ROLE_ALERT, ...(role?.split(' ') || [])])].join(' ')
```

Comment:
There are a bunch of different ways I could have done this, but essentially I am just making sure `alert` doesn't end up in the `role` value twice.

Reply:
Following some a11y investigation I think we were going to remove adding a role if you're loading as it causes invalid aria states anyway. So it might be easier to just get rid of that entirely. @dfmcphee can probably talk more about that.

Reply:
Yep, this is one of the things flagged because alert isn't a valid role for a button. I think we should remove this default behaviour altogether.

<img width="410" alt="Screen Shot 2020-11-03 at 10 15 42 AM" src="https://user-images.githubusercontent.com/478990/98004209-a3db1300-1dbd-11eb-9ac0-9427784eb7c2.png">

Reply:
👍  I will push a new commit that 🔥 the `alert`.

Would we want `alert` on the `<a />` by any chance? Or just kill it entirely?

Reply:
> Would we want `alert` on the `<a />` by any chance? Or just kill it entirely?

I would say kill it entirely. Having a loading link seems like a bit of an anti pattern anyway.

Reply:
Alright folks - new commit has been pushed that removes `alert`!

Reply:
I think you are now getting a fail because it is expecting those two failures listed in the check. You can remove them here: https://github.com/Shopify/polaris-react/blob/master/scripts/accessibility-check.js#L103-L104

Reply:
Excellent 😄  thank you!